### PR TITLE
feat: add `hugWidth` property to the button component which displays it in full width.

### PR DESCRIPTION
**Pull request by juergscherrer**

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -84,6 +84,7 @@ export type ButtonProps = {
     icon?: ReactElement;
     children?: string;
     onClick?: (event?: MouseEvent<HTMLButtonElement>) => void;
+    hugWidth?: boolean;
```
Comment (SamuelAlev) [inline src/components/Button/Button.tsx:87]: We use `hugContent` in the custom blocks, I would sync with designers first

File: src/components/Button/Button.tsx [hunk 2]
```diff
@@ -95,6 +96,7 @@ export const Button: FC<ButtonProps> = ({
     icon,
     children,
     onClick,
+    hugWidth = false,
```
Comment (SamuelAlev) [inline src/components/Button/Button.tsx:99]: It should be `true` by default as we don't want button to be full width by default

File: src/components/Button/Button.tsx [hunk 3]
```diff
@@ -124,6 +126,7 @@ export const Button: FC<ButtonProps> = ({
                           ]
                         : [isFocusVisible && FOCUS_STYLE, styles[getButtonTheme()][style]],
                 ),
+                hugWidth && "tw-w-full",
```
Comment (SamuelAlev) [inline src/components/Button/Button.tsx:129]:
```suggestion
                !hugWidth && "tw-w-full",
```
"Hug" is the current behavior (inline & fit to content)

Comment (juergscherrer) [reply src/components/Button/Button.tsx:129]: Ah sorry, then I misunderstood. Thank you for clarifying.

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**